pydevd_attach_to_process/windows/attach.cpp

```diff
@@ -310,6 +310,11 @@ extern "C"
         // Either _PyThreadState_Current or _PyThreadState_UncheckedGet are required
         DEFINE_PROC_NO_CHECK(curPythonThread, PyThreadState**, "_PyThreadState_Current", -220);  // optional
         DEFINE_PROC_NO_CHECK(getPythonThread, _PyThreadState_UncheckedGet*, "_PyThreadState_UncheckedGet", -230);  // optional
+        DEFINE_PROC_NO_CHECK(getPythonThread13, _PyThreadState_GetCurrent*, "_PyThreadState_GetCurrent", -231);  // optional
```

Comment:
This was necessary to get attach to work in 3.13. There is no `_PyThreadState_UncheckedGet` in 3.13 anymore.